package.json

```diff
@@ -81,7 +81,7 @@
     "babel-preset-react": "^6.24.1",
     "babel-register": "^6.26.0",
     "binary-charts": "^9.2.7",
-    "binary-common-utils": "^7.0.0",
+    "binary-common-utils": "8",
```

Comment:
It's much better to let yarn/npm change this file. I'd prefer `^8.0.0` instead.

Reply:
Done